### DISCUSSION TITLE
How to increase memory for Krita managed server?

### DISCUSSION THREAD

**Discussion by demoshane** [Acly/krita-ai-diffusion#510] [Q&A]
Hey,
everything looks good and promising, but I hit constantly memory issues. Got 32gb and I seem to hit memory issues:

INFO RuntimeError: MPS backend out of memory (MPS allocated: 4.00 GB, other allocations: 1.57 GB, max allowed: 6.77 GB). Tried to allocate 1.52 GB on private pool. Use PYTORCH_MPS_HIGH_WATERMARK_RATIO=0.0 to disable upper limit for memory allocations (may cause system failure).

Is there a way to increase memory allocation (Max allowed) with some settings change (where? wasn't able to find this from plugin dir at least). 

Suggestion: Make GUI setting for max memory alloc.

**Comment by ISkyesI** [comment]
I am not sure if this is what you are looking for, but you can go to the gears icon 
![image](https://github.com/Acly/krita-ai-diffusion/assets/168587807/b591e91f-3772-46ff-9936-29c375736585)
then head to Performance:
![image](https://github.com/Acly/krita-ai-diffusion/assets/168587807/89bc44f5-befb-4816-91a7-69e3c43ddf6e)
And adjust from there.
